Lib/fontTools/misc/plistlib.py

```diff
@@ -378,42 +378,39 @@ def _string_or_data_element(raw_bytes: bytes, ctx: SimpleNamespace) -> etree.Ele
         return _string_element(string, ctx)
 
 
-@singledispatch
-def _make_element(value: Any, ctx: SimpleNamespace) -> NoReturn:
-    raise TypeError("unsupported type: %s" % type(value))
-
-
+# The following is to pacify type checkers. At the time of this writing, neither
+# mypy nor Pyright can deal with singledispatch properly. Mypy additionally
```

Comment:
> neither mypy nor Pyright can deal with singledispatch properly

how about [pytype](https://github.com/google/pytype)?

Reply:
Haven't tried :D

Reply:
is this comment stale? I don't see what is "the following is to pacify typecheckers". What is it unusual about what is following this comment?